static/app/components/organizations/timeRangeSelector/index.tsx

```diff
@@ -1,17 +1,21 @@
 import * as React from 'react';
 import {withRouter, WithRouterProps} from 'react-router';
+import {ClassNames} from '@emotion/react';
 import styled from '@emotion/styled';
 
-import DropdownMenu, {GetActorPropsFn} from 'sentry/components/dropdownMenu';
+import DropdownAutoComplete from 'sentry/components/dropdownAutoComplete';
+import {Item} from 'sentry/components/dropdownAutoComplete/types';
+import {GetActorPropsFn} from 'sentry/components/dropdownMenu';
 import HookOrDefault from 'sentry/components/hookOrDefault';
 import HeaderItem from 'sentry/components/organizations/headerItem';
 import MultipleSelectorSubmitRow from 'sentry/components/organizations/multipleSelectorSubmitRow';
+import PageFilterPinButton from 'sentry/components/organizations/pageFilters/pageFilterPinButton';
 import DateRange from 'sentry/components/organizations/timeRangeSelector/dateRange';
-import SelectorItems from 'sentry/components/organizations/timeRangeSelector/dateRange/selectorItems';
```

Comment:
do we need to delete this old selectorItems?

Reply:
done! https://github.com/getsentry/sentry/pull/32349/commits/ae280483d2cbdebd581a029f6cd5cd1a88b0b0cb